src/define_tables.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Google Inc.
```

Comment:
now it's Google LLC IIRC

Reply:
Thx I'll fix them for the whole code base in a separate patch then update only this one here.